corehq/apps/builds/management/commands/add_commcare_build.py

```diff
@@ -1,26 +1,91 @@
-from django.core.management.base import BaseCommand, CommandError
+import random
+from github import Github
 
-from corehq.apps.builds.models import CommCareBuild
+from django.core.management.base import BaseCommand, CommandError
+from corehq.apps.builds.models import CommCareBuild, CommCareBuildConfig, BuildMenuItem, BuildSpec
 
 
 class Command(BaseCommand):
     help = ('Adds a commcare build, labeled with the version (x.y.z) and build_number (an incrementing integer)\n'
             'to get started see https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/builds/README.md')
 
     def add_arguments(self, parser):
-        parser.add_argument('build_path')
-        parser.add_argument('version')
-        parser.add_argument('build_number')
+        parser.add_argument('build_path', nargs='?')
+        parser.add_argument('version', nargs='?')
+        parser.add_argument('build_number', nargs='?')
+        parser.add_argument(
+            '-l',
+            '--latest',
+            action='store_true',
+            help="add the latest remote CommCare build version"
+        )
 
     def handle(self, build_path, version, build_number, **options):
-        try:
-            build_number = int(build_number)
-        except ValueError:
-            raise CommandError("Build Number %r is not an integer" % build_number)
-
-        try:
-            CommCareBuild.create_from_zip(build_path, version, build_number)
-        except Exception as e:
-            raise CommandError("%s" % e)
-        self.stdout.write('Build %s #%s created\n' % (version, build_number))
-        self.stdout.write('You can see a list of builds at [your-server]/builds/\n')
+        if options.get('latest'):
+            _create_build_from_latest_remote_version()
+        else:
+            if build_path and version and build_number:
+                try:
```

Comment:
Realize you didn't add this code, but fyi you can pass `type=int` to the `add_argument` above.